mousestyles/density_estimation.py

```diff
@@ -0,0 +1,148 @@
+# coding: utf-8
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+
+from mousestyles.data import load_movement
+
+
+def distances(strain, mouse, day, step=50):
```

Comment:
This function is great. I think so far our analysis is on the speed of the mouse. If we want to actually use "distance" instead of "speed",  how to fix the time will be a problem. You function enable us analyze on distance and at the same time the 0 distances. Well, it may also cause some problem. If we choose a small time space, a lot of distances will collapse to 0, while they are not actually zero since the small movement error is not recorded.
Considering this, I think we can set a minimum constrain on step to avoid this problem.
Well, nothing wrong with this function. I just want to make a comment to remind the future users.